### PR TITLE
Fix issue where error "No keyword with name 'No keyword' found.” was

### PR DESCRIPTION
**Pull request by datakurre**
… reported on failing test; Add , which defaults to 'No operation'

### CODE REVIEW

File: buildout.cfg [hunk 1]
```diff
@@ -25,8 +26,16 @@ eggs =
...
+[libdoc]
+recipe = zc.recipe.egg
+eggs = plone.app.robotframework [docs]
+entry-points = libdoc=robot.libdoc:libdoc_cli
+scripts = libdoc
+arguments = sys.argv[1:]
+
 [environment]
 zope_i18n_compile_mo_files = true
+ROBOT_SELENIUM2LIBRARY_RUN_ON_FAILURE = Capture page screenshot and log source
```
Comment (tisto) [inline buildout.cfg]: @datakurre can you elaborate why this is not the default any longer? I first thought, plone.app.robotframework was broken and it took me a while to figure out why there are no screenshots on failures any longer.

Comment (datakurre) [reply buildout.cfg]: I recall Maurits removed it because of some CI issues. I’ll find the pull...

**Comment by datakurre** [comment]
https://github.com/plone/plone.app.robotframework/commit/4ed11100c45351e1488f593da2b61e1b31a19b55

**Comment by datakurre** [comment]
This change came as surprise to me and I had not time to test if that was really the case - even it still sounds strange. That is why I workaround it with a new variable.

**Pull request merged**